templates/_helpers.yaml

```diff
@@ -90,6 +90,15 @@
         key: connection
     {{- end }}
   {{- end }}
+  {{- if semverCompare ">=3.0.0" .Values.airflowVersion }}
```

Comment:
Is this going to work? We need to make sure this is comparing the **airflow** version since runtime does not use semver. As long as `airflowVersion` is actually airflow and not runtime then this should be ok.

Reply:
we are using this chart for our current testing and works as expected since we pass the actual airflow version in this fields